### PR TITLE
Partial rewrite/expansion of `Vec::truncate` documentation.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: src/libcollections/vec.rs [hunk 1]
```diff
@@ -479,18 +479,45 @@ impl<T> Vec<T> {
...
     ///
     /// If `len` is greater than the vector's current length, this has no
     /// effect.
     ///
+    /// The [`drain`] method can emulate `truncate`, but causes the excess
+    /// elements to be returned instead of dropped.
+    ///
     /// # Examples
     ///
+    /// Truncating a five element vector to two elements:
```
Comment (GuillaumeGomez) [inline src/libcollections/vec.rs]: "five element" -> "five elements"?

**Comment by GuillaumeGomez** [comment]
Just a little typo when we're good! :)

File: src/libcollections/vec.rs (hunk 1 above)
Comment (frewsxcv) [inline src/libcollections/vec.rs]: I think "five element" is right. I asked my fiancée and she agreed with me. I wish I knew the exact English rules regarding this, but I don't. Probably something to do with plurality of [attribute nouns](https://en.wikipedia.org/wiki/Noun_adjunct).

Comment (steveklabnik) [inline src/libcollections/vec.rs]: Yes, "five element" is accurate.

**Pull request merged**